avocado/utils/configure_network.py

```diff
@@ -191,3 +191,15 @@ def get_peer_interface(self, peer_ip):
                 log.error("unable to get peer interface: %s", ex)
         else:
             return peer_interface
+
+
+def interface_link_up(interface):
+    '''
+    Checks if the interface link is up
+    '''
+    if 'UP' or 'yes' in \
+        process.system_output("ethtool %s | tail -1"
```

Comment:
can we depend on sysfs attibute example can be 

`if up in '/sys/class/net/%s/operstate' %% interface`